synapse/storage/event_push_actions.py

```diff
@@ -64,6 +64,27 @@ def _deserialize_action(actions, is_highlight):
 
 
 class EventPushActionsWorkerStore(SQLBaseStore):
+    def __init__(self, db_conn, hs):
+        super(EventPushActionsWorkerStore, self).__init__(db_conn, hs)
+
+        # These get correctly ste by _find_stream_orderings_for_times_txn
+        self.stream_ordering_month_ago = 0
+        self.stream_ordering_day_ago = 0
```

Comment:
we have code [here](https://github.com/matrix-org/synapse/pull/2923/files#diff-c464372cc58591010e903cfa4f1c48f2R739) which appears to expect this to be initialised to None (though I see no evidence of it being intialised at all, so perhaps it was just broken before)

Reply:
Ah, may as well set it to `None`. I think before we didn't call `_find_stream_orderings_for_times_txn` on start up whereas now we do, so that code won't be hit currently.